.github/workflows/release.yml

```
@@ -13,7 +14,58 @@ env:
   IMAGE_NAME: ${{ github.repository }}
 
 jobs:
+  test-kubo:
+    name: 'test:kubo'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: L1-node
+
+      - name: Checkout Kubo fork
```

```suggestion
      - name: Check out Kubo fork
```

nit: afaik "checkout" isn't a verb